modules/serverless-functions-secrets-configmaps-alternative-invocations.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Alternative ways to invoke the interactive prompt
+
+There is a number of more specific invocations of the `func config` utility. Use them to skip steps when navigating the `func config` operation tree:
+
+* `func config envs`
```

Comment:
Can you elaborate on this, is this a command? What does it do? E.g. "You can use the following commands to do x" kinda thing?

Reply:
This was sort of a section name for a subset of commands. I've removed these section names altogether, because they don't add much value. Now it's a flat list.